gslab_scons/log.py

```diff
@@ -52,7 +53,7 @@ def end_log(log = 'sconstruct.log'):
             with open(f, 'rU') as sconscript:
                 if this_run_dict[f] == beginning_of_time:
                     sconstruct.write("*** Warning!!! Doesn't look the sconscript below finished.\n")
```

Comment:
Sure thing. I assumed you wanted to keep the new line at the end, but let me know if not.